File: Scripts/Outros/ADS/Java/501/aulas/03.aul23/Empresa/Funcionario.php
<?php
  //criando classe funcionario
    class Funcionario
    {
      //criando atributos
        public $nome, $cpf, $salario;

      //criando funcao mostrar dados
        public function mostrarInformacoes()
        {
          //retornando valores
            return " Nome: ".$this->nome ." Cpf: ".$this->cpf ." Salario: ".$this->salario;
        }
    }

?>

File: Scripts/Outros/ADS/Java/201/Aula01_12/src/Funcionario.java

public class Funcionario {
    String Nome;
    String Cargo;
    double Salario;
    
    Funcionario(String Nome, String Cargo, Double Salario)
    {
        this.Nome = Nome;
        this.Salario = Salario;
        this.Cargo = Cargo;
    }
    
    void Programar(String Sistema)
    {
        System.out.println("Programando o sistema " + Sistema + " em " + this.Cargo);
    }
}
